### PR TITLE
Add capistrano and target login.gov domain lower envs

### PR DESCRIPTION
**Pull request by jgrevich**
Ready to 🎢

### CODE REVIEW

**Comment by amoose** [comment]
My only tweak is to change the domain from rp.login.gov to sp-sinatra.login.gov.

**Comment by pkarman** [comment]
Can you add line or two (or amend) what is in README? Is it `cap demo deploy` e.g.?

**Comment by jgrevich** [comment]
> My only tweak is to change the domain from rp.login.gov to sp-sinatra.login.gov.

I thought I changed that already. I didn't have a cert at the time for sp-sinatra. I'll have the fix right quick.

**Comment by jgrevich** [comment]
> Can you add line or two (or amend) what is in README? Is it cap demo deploy e.g.?

Yep, coming up.

**Comment by jgrevich** [comment]
Ready to 🎢  take 2

**Comment by pkarman** [comment]
lgtm % failing travis

**Comment by jgrevich** [comment]
Go figure, the third time is the charm. Ready to 🎢

**Comment by amoose** [comment]
GTG 🚢

**Comment by amoose** [comment]
This requires an update to the IdP, I will PR with the updated endpoints

File: config/saml_settings_dev.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
-assertion_consumer_service_url: https://identity-rp-dev.apps.cloud.gov/consume
+assertion_consumer_service_url: https://sp-sinatra.dev.login.gov/consume
 assertion_consumer_service_binding: urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST
-issuer: urn:gov:gsa:SAML:2.0.profiles:sp:sso:dev
-idp_sso_target_url: https://upaya-idp-dev.18f.gov/api/saml/auth
-idp_slo_target_url: https://upaya-idp-dev.18f.gov/api/saml/logout
+issuer: urn:gov:gsa:SAML:2.0.profiles:sp:sso:demo
```
Comment (amoose) [inline config/saml_settings_dev.yml]: Just noticed this should be `dev`, will PR shortly

Comment (amoose) [inline config/saml_settings_dev.yml]: https://github.com/18F/identity-sp-sinatra/pull/32

**Pull request merged**